sns/parity_math.c

```diff
@@ -229,16 +229,22 @@ static void dependency_bitmap_prepare(struct m0_sns_ir_block *f_block,
 
 #if ISAL_ENCODE_ENABLED
 /**
- * Converts m0_bufvec to m0_buf and vice versa.
- * @param bvec  - Pointer to buffer vector.
- * @param buf   - Pointer to buffer.
- * @param count - Number of buffers to copy
- * @param dir   - Decides direction of conversion.
- *                True  - Copy data from buffer vector to buffer
- *                False - Copy data from buffer to buffer vector
+ * Converts m0_bufvec to m0_buf.
+ * @param bvec[in]  - Pointer to buffer vector.
+ * @param buf[out]  - Pointer to buffer.
```

Comment:
This is an array of pointer, not a single pointer, right?
How do we make sure the "buf" array has enough elements?
How do we make sure the i-th "buf" has enough memory?

Reply:
Updated functions bufvec_to_buf_copy() and buf_to_bufvec_copy().